library-front/src/components/pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,31 @@
+.home-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  padding-top: 100px;
+  padding-bottom: 100px;
+}
+
+.home-container > .BookCard {
+  padding: 10px;
+  margin: 5px;
+}
+
+@media only screen and (min-width: 769px) {
+  .home-container {
```

Comment:
It looks like there's some redundancy in the CSS code. We don't need to set the same properties for both desktop and mobile because they should be already applied. For example, if we use display: flex; it will be applied to both desktop and mobile devices.

However, if we want to change a property for a specific device, we need to set it again. This approach will help us keep the code lean and avoid unnecessary repetition.